docker-compose.yml

```diff
@@ -11,6 +10,19 @@ services:
     restart: always
     profiles:
       - "hello-service"
+  jenkins:
+    image: jenkins/jenkins
+    ports:
+      - 8082:8080
+      - 50000:50000
+    restart: on-failure
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /usr/bin/docker:/usr/bin/docker
+      - ./Dockerfile:/Dockerfile
+    profiles:
+      - "hello-service"
```

Comment:
Does this mean we always start Jenkins?

Reply:
Yes. The Jenkins container does not do anything until it is activated from the UI. And between this and making another `.sh` file, I opted for the former.